Remotion/Development/Web/UnitTesting/ExecutionEngine/WxeFunctionThreadAbortHelper.cs

```diff
@@ -7,19 +7,12 @@ public static class WxeThreadAbortHelper
   {
     public static void Abort ()
     {
-#if NETFRAMEWORK
-      Thread.CurrentThread.Abort();
-#else
       var exception = (ThreadAbortException)Activator.CreateInstance(typeof(ThreadAbortException), nonPublic: true)!;
       throw exception;
-#endif
     }
 
     public static void ResetAbort ()
```

Comment:
delete please. should be a separate commit since the tests get touched by this and we should verify the correctness. we could remove in a separate task/PR, if you want.

Reply:
The whole file or just the reset abort?

Reply:
the RestAbort method since it's blank